src/Microsoft.AspNet.Mvc.Core/Filters/AuthorizeFilter.cs

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Security;
+using Microsoft.Framework.DependencyInjection;
+
+namespace Microsoft.AspNet.Mvc
+{
+    // REVIEW: figure out how to propery rationalize this with AuthorizationFilterAttribute
+    public class AuthorizeFilter : IAsyncAuthorizationFilter, IOrderedFilter
+    {
+        public int Order { get; set; }
+
+        public string Policy { get; set; }
+
+        public IEnumerable<string> Roles { get; set; }
```

Comment:
I don't really grok this comment or the response. I don't see a `ToArray()` so I'm assuming there's no issue here?